website/docs/cli/commands/test/index.mdx

```diff
@@ -0,0 +1,368 @@
+import ProviderAliasTest from '!!raw-loader!./examples/provider_alias/main.tftest.hcl'
+import VariablesMain from '!!raw-loader!./examples/variables/main.tf'
+import VariablesTest from '!!raw-loader!./examples/variables/main.tftest.hcl'
+import ExpectFailureVariablesMain from '!!raw-loader!./examples/expect_failures_variables/main.tf'
+import ExpectFailureVariablesTest from '!!raw-loader!./examples/expect_failures_variables/main.tftest.hcl'
+import ExpectFailureResourcesMain from '!!raw-loader!./examples/expect_failures_resources/main.tf'
+import ExpectFailureResourcesTest from '!!raw-loader!./examples/expect_failures_resources/main.tftest.hcl'
+
+# Command: test
+
+The `tofu test` command lets you test your OpenTofu configuration by creating real infrastructure and checking that
+the required conditions (assertions) are met. Once the test is complete, OpenTofu destroys the resources it created.
+
+## Usage
+
+Usage: `tofu test [options]`.
+
+This command will execute all `*.tftest.hcl` files in the current directory or in a directory called `tests`. You can
+customize this behavior using the [options](#options) below.
+
+:::info Example
+
+Consider the following simple example which creates a `test.txt` file from `main.tf` and then checks that the main code
+has successfully performed its job from `main.tftest.hcl`.
+
+<Tabs>
+    <TabItem label={"main.tf"} value={"main"} default>
+        <CodeBlock language="hcl">{SimpleMain}</CodeBlock>
+    </TabItem>
+    <TabItem label={"main.tftest.hcl"} value={"test"}>
+        <CodeBlock language="hcl">{SimpleTest}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+You can run `tofu init` followed by `tofu test` to execute the test which will apply the `main.tf` file and test it
+against the assertion in `main.tftest.hcl`. This is just a simple illustration. You can find more comprehensive examples
+below.
+
+:::
+
+## Options
+
+* `-test-directory=path` Set the test directory (default: "tests"). OpenTofu will search for test files in the specified
+  directory and also the current directory when you run `tofu test`. The path should be relative to the current
+  working directory.
+* `-filter=testfile` Specify an individual test file to run. Use this option multiple times to specify more than one
+  file. The path should be relative to the current working directory.
+* `-var 'foo=bar'` Set an input variable of the root module. Specify this option multiple times to add more
+  than one variable.
+* `-var-file=filename` Set multiple variables from the specified file. In addition to this file, OpenTofu automatically
+  loads `terraform.tfvars` and `*.auto.tfvars`. Use this option multiple times to specify more than one file.
+* `-json` Change the output format to JSON.
+* `-no-color` Disable colorized output in the command output.
+* `-verbose` Print the plan or state for each test run block as it executes.
+
+## Directory structure
+
+The `tofu test` command supports two directory layouts:
```

Comment:
```suggestion
The `tofu test` command supports two directory layouts, flat or nested:
```
This helps clarify things a bit more by being explicit, instead of assuming the tab titles are a continuation of your sentence.